theme/themes/eea/elements/list.variables

```diff
@@ -181,12 +177,13 @@
 
 /* Ordered List */
 @orderedCountName: ordered;
-@orderedCountContent: counters(ordered, ".") " ";
+@orderedCountContent: counters(ordered, '.') ' ';
 @orderedCountColor: inherit;
 @orderedCountDistance: 1.25rem;
 @orderedCountOpacity: 1;
 @orderedCountTextAlign: right;
 @orderedCountVerticalAlign: middle;
+@orderedDistance: @rem-space-5;
```

Comment:
@tedw87 set value of @orderedDistance: @bulletDistance;
This way is that variable is different this value will take that value as
the right value to set for the padding.